scipy/special/setup.py

```diff
@@ -13,6 +13,20 @@
                      (numpy.__version__, numpy.__file__)) from e
 
 
+def cxx_pre_build_hook(build_ext, ext):
+    from scipy._build_utils.compiler_helper import (get_cxx_std_flag,
+                                                    try_add_flag)
+    cc = build_ext._cxx_compiler
+    args = ext.extra_compile_args
+
+    std_flag = get_cxx_std_flag(cc)
+    if std_flag is not None:
+        args.append(std_flag)
+    if sys.platform == 'darwin':
+        args.append('-mmacosx-version-min=10.7')
+        try_add_flag(args, cc, '-stdlib=libc++')
```

Comment:
This `libc++` is the default since OSX 10.9, so this line can be removed. It's not needed anywhere else in SciPy, and we have C++11 code in other places.